src/main/drivers/max7456.c

```diff
@@ -212,7 +212,7 @@ static uint8_t shadowBuffer[VIDEO_BUFFER_CHARS_PAL];
 volatile bool dmaTransactionInProgress = false;
 #endif
 
-static uint8_t spiBuff[MAX_CHARS2UPDATE*6];
+static uint8_t spiBuff[MAX_CHARS2UPDATE*6+40]; // Took the extra 40 bytes from screenBuffer
```

Comment:
That comment is not helpful at all, since the original comment will be gone. ;-)